src/components/info/byline.js

```diff
@@ -106,11 +109,13 @@ function renderMaintainers(t, metadata) {
  * Returns a React component detailing the date the data was last updated.
  * Renders a <span> containing "Data updated X", where X derives from `metadata.updated`
  */
-function renderDataUpdated(t, metadata) {
-  if (metadata.updated) {
+function renderDataUpdated(t, metadata, language) {
+  if (moment(metadata.updated).isValid()) {
     return (
       <span>
-        {`${t("Data updated")} ${metadata.updated}. `}
+        {`${t("Data updated")} `}
+        <Moment locale={language} date={metadata.updated} fromNow />
```

Comment:
One scenario where this change would be unfavorable is in screenshots, where relative dates don't age well. (I just noticed this while watching someone present their Nextstrain build).

Reply:
Displaying both would make things more interpretable without compromising screenshots, but maybe that's getting too wordy?

> Data updated 3 years ago (2020-12-11)

Reply:
Personally, I get irked by relative dates without the ability to see the exact date. I think displaying both would be better.